strings_lab.js

```diff
@@ -0,0 +1,255 @@
+// # String Exercises
+
+// 1. `console.log` the first character of a string.
+console.log("#1 ---------------------\n");
+
+let str = "This is the best string ever coded";
+console.log(str[0]);
+console.log(" ");
+
+// 2. `console.log` the length of a string.
+console.log("#2 ---------------------\n");
+
+console.log(str.length);
+console.log(" ");
+
+// 3. `console.log` the last character of any string.
+console.log("#3 ---------------------\n");
+
+console.log(str[str.length - 1]);
+console.log(" ");
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+console.log("#4 ---------------------\n");
+
+// let name = "alejo";
+// let upperCased = name.toUpperCase();
+// console.log("upperCased: " + upperCased);
+// let lowerCased = upperCased.toLowerCase();
+// console.log("lowerCased: " + lowerCased);
+
+let name = "doug";
```

Comment:
Would this code still work if I change name to "Alejandro"? by setting (slice(0, 3)), you are hard coding as it cannot accommodate for different string.